libs/stocks/feature-shell/src/lib/stocks/stocks.component.ts

```diff
@@ -32,7 +32,9 @@ export class StocksComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.stockPickerForm.valueChanges.subscribe(this.fetchQuote);
```

Comment:
To avoid 'scope' issue inside the 'fetchQuote' method. call ' this.fetchQuote' inside arrow function.

eg: 
this.form.subscribe(()=>{
   // call method
});